PrimerKlarnaSDK/PrimerKlarnaSDK/Sources/Wrapper/PrimerKlarnaWrapper.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  PrimerKlarnaWrapper.swift
+//  PrimerKlarnaSDK
+//
+//  Created by Illia Khrypunov on 30.10.2023.
+//
+
+import Foundation
+import KlarnaMobileSDK
+
+// MARK: - PrimerKlarnaWrapper
+public protocol PrimerKlarnaWrapper: AnyObject {
```

Comment:
What do you think about the following naming convention?:

protocol: `PrimerKlarnaProviding`
impl: `PrimerKlarnaWrapper`, or simply `PrimerKlarna`

Reply:
Maybe 
protocol: PrimerKlarnaProviding
impl: PrimerKlarnaProvider